Honor color/opacity/width of track/route as specified in GPX

Bohusz opened the pull request:
Čau Vyldo,

zdá se, že nejnovější změny jsou ve větvi **master_Vylda**, čekal bych je v **master**.

Bohusz committed:
Honor color/opacity/width of track/route as specified in GPX

Jsou podporovány tři způsoby specifikace

1. Locus Line Extensions
   - extensions > line > extensions > locus:lsColorBase
   - extensions > line > extensions > locus:lsWidth
2. Topografix Line Extensions
   - extensions > line > color
   - extensions > line > opacity
   - extensions > line > width
3. Garmin Route Extensions
   - extensions > gpxx:RouteExtension > gpxx:DisplayColor

Hladají se v uvedeném pořadí (první nalezená se použije)
v elementech
- gpx > trk
- gpx > trk > trkseg
- gpx > rte

Vylda commented:
No, možná by bylo fajn poslat na pull request fakt jen to, co jsi změnil, protože kdo má číst ty kvanta změn, kde je změna jen v odsazení.

Bohusz committed:
Honor color/opacity/width - oprava odsazení

Rozsáhlé části skripty bylo oproti původnímu kódu
odsazena o jeden tabelítor, některé části o dva.

Nyní vráceno do původního stavu.

Vylda commented on GeocachingMapEnhancements/GeocachingMapEnhancements.user.js:
```diff
@@ -955,8 +958,53 @@
+							}
 							function parseGPX(text) {
-								var d, i, j, k, w, r, t, lat, lng, len, n, name = "", poly, type, points = { primary: [], additional: [], routes: [] }, gpx = $($.parseXML(text)), wpts = gpx.find("wpt"), tracks = gpx.find("trk"), segs, routes = gpx.find("rte");
+							var d, i, j, k, w, r, t, lat, lng, len, n, opts, name = "", poly, type, points = { primary: [], additional: [], routes: [] }, gpx = $($.parseXML(text)), wpts = gpx.find("wpt"), tracks = gpx.find("trk"), segs, routes = gpx.find("rte");
```
```suggestion
							var d, i, j, k, w, r, t, lat, lng, len, n, name = "", opts, poly, type, points = { primary: [], additional: [], routes: [] }, gpx = $($.parseXML(text)), wpts = gpx.find("wpt"), tracks = gpx.find("trk"), segs, routes = gpx.find("rte");
```

Vylda commented on GeocachingMapEnhancements/GeocachingMapEnhancements.user.js:
```diff
@@ -984,27 +1032,30 @@
 										poly.push(new L.LatLng(w.attr("lat"), w.attr("lon")));
-									}
-									points.routes.push({ name: name, points: poly });
+								}
```
```suggestion
									}
```

Vylda commented on GeocachingMapEnhancements/GeocachingMapEnhancements.user.js:
```diff
@@ -1792,7 +1843,7 @@
 
 					/* If we're adding our own map selector control, we need to manually remove any pre-existing map layers.  Otherwise, they persist in the background underneath
-					 * the layers provided by GME.	We check for the _url or _google attribute to distinguish map layers from other Leaflet layers like controls or popups */
+                        * the layers provided by GME.	We check for the _url or _google attribute to distinguish map layers from other Leaflet layers like controls or popups */
```
```suggestion
                      * the layers provided by GME.	We check for the _url or _google attribute to distinguish map layers from other Leaflet layers like controls or popups */
```

Vylda commented on GeocachingMapEnhancements/GeocachingMapEnhancements.user.js:
```diff
@@ -1993,7 +2044,7 @@
 						options: { position: "bottomleft" },
 						onAdd: function (contextmap) {
-							var elem, container = L.DomUtil.create("div", "leaflet-control-gme"), control = this, html = "";
+						var elem, container = L.DomUtil.create("div", "leaflet-control-gme"), control = this, html = "";
```
```suggestion
							var elem, container = L.DomUtil.create("div", "leaflet-control-gme"), control = this, html = "";
```

Vylda reviewed (commented):
Pár drobností, rádky, do kterých jsi nesahal bys neměl měnit.
